pull/github_test.go

```diff
@@ -396,6 +396,35 @@ func TestIsOrgMember(t *testing.T) {
 	assert.Equal(t, 1, yesRule.Count, "cached membership was not used")
 }
 
+func TestBranches(t *testing.T) {
+	rp := &ResponsePlayer{}
+	ctx := makeContext(t, rp, nil)
+
+	base, head := ctx.Branches()
+	assert.Equal(t, "develop", base, "base branch was not correctly set")
+	assert.Equal(t, "test-branch", head, "head branch was not correctly set")
+}
+
+func TestCrossRepoBranches(t *testing.T) {
+	rp := &ResponsePlayer{}
+
+	// change the source repo to a forked repo
+	crossRepoPr := defaultTestPR()
+	crossRepoPr.Head.Repo = &github.Repository{
+		ID: github.Int64(12345),
+		Owner: &github.User{
+			Login: github.String("testorg2"),
+		},
+		Name: github.String("testrepofork"),
+	}
+
+	ctx := makeContext(t, rp, crossRepoPr)
+
+	base, head := ctx.Branches()
+	assert.Equal(t, "develop", base, "cross-repo base branch was not correctly set")
+	assert.Equal(t, "testorg2:test-branch", head, "cross-repo head branch was not correctly set")
```

Comment:
The documentation for this predicate should probably mention the fact that source branches for PRs from forks will have a `:` and that patterns will need to account for that.

Alternately, we could strip the organization from this name and then add a separate `from_fork` predicate for policies that want to differentiate forks. Right now, `from_branch` serves both purposes and a pattern like `:` would work as a "is this a fork" condition.

Reply:
I left this together and added that info to the readme. Seemed like a decent solution to me. Let me know if you feel strongly about splitting them out and I can.